composer.json

```diff
@@ -40,7 +40,8 @@
         "sumocoders/framework-search-bundle": "^3.0",
         "sumocoders/framework-example-bundle": "^4.0",
         "sumocoders/framework-multi-user-bundle": "^4.0",
-        "knplabs/knp-paginator-bundle": "^2.5"
+        "knplabs/knp-paginator-bundle": "^2.5",
+        "simple-bus/symfony-bridge": "^4.1"
```

Comment:
I don't remember in exactly what situations but in some if you don't have this the deploys fail, it is also for that reason included in fork